src/util-base64.c

```diff
@@ -63,6 +63,22 @@ static inline int GetBase64Value(uint8_t c)
     return val;
 }
 
+/**
+ * \brief Checks if the given char in a byte array is Base64 alphabet
+ *
+ * \param src Pointer to the byte array
+ * \param idx Index where to look at
+ *
+ * \return True if the char was Base64 alphabet, False otherwise
+ */
+bool IsBase64Alphabet(const uint8_t *src, const uint32_t idx)
```

Comment:
why not pass in the char itself?

Reply:
For some reason, thought this was the C way. 😭  nvm. Will do it in the next rev. Thank you!